packages/app-mobile/components/voiceTyping/AudioRecordingBanner.tsx

```
@@ -239,4 +223,4 @@ const AudioRecordingBanner: React.FC<Props> = props => {
 	/>;
 };
 
-export default AudioRecordingBanner;
+export default AudioRecordingBanner;
```

ESLint

[error] Newline required at end of file but not found.

(eol-last)